hatchet_sdk/worker/runner/run_loop_manager.py

```diff
@@ -52,6 +52,7 @@ async def _async_start(self, retry_count=1):
         logger.info("starting runner...")
         self.loop = asyncio.get_running_loop()
         k = self.loop.create_task(self._start_action_loop())
+        await k
```

Comment:
this is required to keep the logging interceptor running, and doesnt seem to have any side effects but sanity check here would be helpful